biohansel/utils.py

```diff
@@ -1,23 +1,27 @@
 # -*- coding: utf-8 -*-
-
-import logging
-from typing import List, Any, Optional, Tuple, Union
-
+"""
+Common utility functions and constants. 
+"""
+from typing import List, Any, Tuple, Union
 import os
 import re
+import logging
 from collections import defaultdict
 
-from .const import SCHEME_FASTAS, REGEX_FASTQ, REGEX_FASTA
-from .subtyping_params import SubtypingParams
+LOG_FORMAT = '%(asctime)s %(levelname)s: %(message)s [in %(pathname)s:%(lineno)d]'
+
+NT_SUB = {x: y for x, y in zip('acgtrymkswhbvdnxACGTRYMKSWHBVDNX', 'tgcayrkmswdvbhnxTGCAYRKMSWDVBHNX')}
```

Comment:
just a small thing, but perhaps a more descriptive variable name?

Reply:
`NT_SUB` is const nucleotide substitution dict for reverse complementing a nucleotide sequence. It's not really meant to be used by anything except the `revcomp` function. It's in the module scope so it isn't re-initialized each time the `revcomp` function is called.

Reply:
Changed in 47fa20fae518aee051462ab5c9116422d5bd47db. Added a comment as well.